test/disability-benefits/components/StatusPage.unit.spec.jsx

```diff
@@ -101,4 +102,59 @@ describe('<StatusPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
+  it('should render notification', () => {
+    const claim = {};
+
+    const tree = SkinDeep.shallowRender(
+      <StatusPage
+          loading
+          message={{ title: 'Test', body: 'Body' }}
+          claim={claim}/>
+    );
+    expect(tree.props.message).not.to.be.null;
```

Comment:
Actually, this is correct. ClaimDetailLayout is the root component, so tree.props is checking the props for that component, which is what I'm intending.